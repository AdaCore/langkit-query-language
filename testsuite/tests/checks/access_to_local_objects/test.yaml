driver: 'checker'
project: 'checks/checks.gpr'
rule_name: Access_To_Local_Objects