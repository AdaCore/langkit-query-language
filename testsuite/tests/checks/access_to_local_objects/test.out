access_to_local_objects.adb:8:32: error: access_to_local_objects - rule violation
8 |        Var2 :         Int_A := Var1'Access;  --  FLAG
  |                                ^^^^^^^^^^^ 

access_to_local_objects.adb:18:24: error: access_to_local_objects - rule violation
18 |        Var5 : Int_A := Var4'Access;   -- FLAG
   |                        ^^^^^^^^^^^ 

access_to_local_objects.adb:32:25: error: access_to_local_objects - rule violation
32 |            P : Int_A := Local'Access; -- FLAG
   |                         ^^^^^^^^^^^^ 

